lib/src/presentation/view/card_page.dart

```diff
@@ -69,69 +67,101 @@ class _CardPageState extends State<CardPage> {
+                                widget.hearthstoneCard.mechanics!.isNotEmpty,
+                            child: Row(
+                              mainAxisSize: MainAxisSize.min,
+                              crossAxisAlignment: CrossAxisAlignment.start,
+                              children: [
+                                Padding(
+                                  padding: const EdgeInsets.only(
+                                    top: cardPageListViewPaddingTop,
+                                  ),
+                                  child: VisibilityText(
+                                    cardAttribute: cardMechanics,
+                                    isVisible: widget
+                                        .hearthstoneCard.mechanics!.isNotEmpty,
+                                  ),
+                                ),
+                                Expanded(
+                                  child: ListView.builder(
+                                    padding: const EdgeInsets.all(
+                                      cardListPaddingListView,
+                                    ),
+                                    shrinkWrap: true,
+                                    physics:
+                                        const NeverScrollableScrollPhysics(),
+                                    itemCount: widget
+                                        .hearthstoneCard.mechanics!.length,
+                                    itemBuilder: (
+                                      BuildContext context,
+                                      index,
+                                    ) {
+                                      return VisibilityText(
+                                        cardAttribute:
+                                            ' ${widget.hearthstoneCard.mechanics![index]}',
+                                        isVisible: widget.hearthstoneCard
+                                            .mechanics!.isNotEmpty,
+                                      );
+                                    },
+                                  ),
+                                ),
+                              ],
+                            ),
                           ),
+                          SizedBox(
+                            width: cardImageBoxSizeWidth,
+                            height: cardImageBoxSizeHeight,
+                            child: FittedBox(
+                              fit: BoxFit.fill,
+                              child: widget.hearthstoneCard.img != null
+                                  ? Image.network(
+                                      widget.hearthstoneCard.img!,
+                                      fit: BoxFit.fill,
+                                    )
+                                  : Image.asset(
+                                      assetCardBackImage,
+                                    ),
+                            ),
```

Comment:
You can abstract this in a new widget, you are using the same in card_list

Reply:
Done.